book/embeds.md

```diff
@@ -2231,14 +2215,15 @@ used to [construct a clock][sab-attack].
 
 ::: {.further}
 The `SharedArrayBuffer` issue caused problems when I [added JavaScript
-support][js-blog] to the embedded browser widgets on this website. I
+support][js-blog] to the embedded browser widgets on
+[the book's website](https://browser.engineering). I
 was using `SharedArrayBuffer` to allow synchronous calls from a
 `JSContext` to the browser, and that required APIs that browsers
 restrict for security reasons. Setting the security headers wouldn't
-work, because [Chapter 14](accessibility.md) embeds a Youtube video,
```

Comment:
I kept the links. I think we'll strip them out with Lua for LaTeX and keep them for HTML.

Reply:
In this case I didn't think it really adds value to the web version, so removed it.